addons/crm/views/crm_lead_views.xml

```diff
@@ -576,47 +576,114 @@
             </field>
         </record>
 
+        <act_window name="Send email"
+                res_model="mail.compose.message"
```

Comment:
HumanLinter: you have indentation issues (indentation should always be 4 spaces) -> be sure to configure your linter accordingly